src/assets/css/layouts/header.css

```diff
@@ -13,7 +13,7 @@
   gap: 10px;
 }
 .header_logo_img {
-  padding-left: 46px;
+  padding-left: 146px;
```

Comment:
Why do we have a large number here?

Reply:
Yesterday I must have mistakenly deleted the number 1 while coding